dbt_invoke/properties.py

```diff
@@ -445,6 +447,32 @@ def _delete_all_property_files(ctx, transformed_ls_results):
     else:
         _LOGGER.info('There are no files to delete.')
 
+@functools.lru_cache
+def _read_manifest(
+    target_path
+):
+    with open(Path(target_path,'manifest').with_suffix('.json'), "r",encoding='utf-8') as manifest_json:
+        return json.loads(manifest_json.read())
+
+def _get_current_location(
+    ctx,
+    resource_dict
+):
+    """
+    Find if the resource is already documented somewhere
+
+    :param ctx: An Invoke context object
+    :param resource_dict: A dictionary representing the json output for
+        this resource from the "dbt ls" command
+    :return: Str or None
+    """
+    node_unique_id = resource_dict['unique_id']
+    patch_path = _read_manifest(ctx.config['target_path'])['nodes'][node_unique_id]['patch_path']
```

Comment:
@robastel this section of the manifest is the result of dbt having discovered the matching yaml for that node.

Reply:
It is created when the dbt compile takes place from memory.

Reply:
Ah yes, great, I forgot you already did that!

Logistically, how would you like to go about working on our hybrid approach?

I'm happy to take an attempt at combining our two approaches. Also happy to review if you feel like you want to take a shot at it. I'm going to sign off for today, but I'll keep in touch!

Reply:
I have finished work for the week now. Happy to take a stab at this Monday if you haven’t already done it. 🙂